src/app/dataValidator/helpers/validators/validateNode.test.js

```diff
@@ -143,30 +143,34 @@ describe('Validate block', () => {
     const paragraphBlock = textBlocks[0];
     const paragraphBlocks = paragraphBlock.model.blocks;
     const fragmentBlock = paragraphBlocks[0];
+    const textAttributes = fragmentBlock.model.attributes;
```

Comment:
It'd be best to call these `fragmentAttributes`, since these attributes are related to the fragment block.